src/DropDownIcon.jsx

```diff
@@ -60,7 +60,7 @@ class DropDownIcon extends Component {
           anchorEl={this.state.anchorEl}
           anchorOrigin={{horizontal: 'left', vertical: 'top'}}
           onRequestClose={this._handleRequestClose}
-          pushToLeft={this.props.pushToLeft}
+          rePositionToBottomRight={this.props.rePositionToBottomRight}
```

Comment:
reposition is one word so it should be repositionToBottomRight rather than rePosition...